tests/unit_tests/test_wave_06.py

```diff
@@ -76,16 +76,28 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That the results is truthy
+    assert result
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
     # - That all the correct items are in tai and jesse's inventories, including the items which were swapped from one vendor to the other
+    assert item_a in tai.inventory
```

Comment:
Nice use of `in` since the specification doesn't stipulate where the swapped items will end up in the output!